src/Symfony/Component/Validator/Resources/translations/validators.fr.xlf

```diff
@@ -440,7 +440,7 @@
             </trans-unit>
             <trans-unit id="113">
                 <source>This URL is missing a top-level domain.</source>
-                <target state="needs-review-translation">Cette URL est dépourvue de domaine de premier niveau.</target>
+                <target>Cette URL est dépourvue de domaine de premier niveau.</target>
```

Comment:
```suggestion
                <target>Cette URL ne contient pas de domaine de premier niveau.</target>
```
Voire
```suggestion
                <target>Cette URL doit contenir un domaine de premier niveau.</target>
```


It feels to me "dépourvue" add some reading complexity 

WDYT ?

Reply:
I was wondering, and looked at other "is missing" usage but there is none...

Looking at other strings, they are either
* _imperative_ like "doit être", "ne doit pas être" which suits better "doit contenir"
* _affirmative_ like "n'est pas", "n'a pas" which suits better "ne contient pas"

I am not sure which one to choose here, if it's only personal I'd go with "doit contenir" which sounds natural for me - avoiding a negative form on the way. That's also why I left "est dépourvue" but I can understand it's not a word we are used to seeing in error.

Maybe having other opinions on that question can help to choose the best one.